pkg/identity/cache/local.go

```diff
@@ -194,14 +194,10 @@ func (l *localIdentityCache) GetIdentities() map[identity.NumericIdentity]*ident
 	return cache
 }
 
-// close closes the events channel. The local identity cache is the writing
-// party, hence also needs to close the channel.
+// close removes the events channel.
 func (l *localIdentityCache) close() {
 	l.mutex.Lock()
 	defer l.mutex.Unlock()
 
-	if l.events != nil {
-		close(l.events)
-		l.events = nil
-	}
+	l.events = nil
```

Comment:
Why no channel close here? Seems like only `localIdentityCache` uses this channel to send.

Reply:
right; now there are two caches that send events to the same channel. So, we have to move the channel close "up a level" to prevent a double-close.